target/linux/mediatek/image/filogic.mk

```diff
@@ -453,6 +453,25 @@ define Device/confiabits_mt7981
 endef
 TARGET_DEVICES += confiabits_mt7981
 
+define Device/cudy_re3000-v1
+  DEVICE_VENDOR := Cudy
+  DEVICE_MODEL := RE3000
+  DEVICE_VARIANT := v1
+  DEVICE_DTS := mt7981b-cudy-re3000-v1
+  DEVICE_DTS_DIR := ../dts
+  DEVICE_DTS_LOADADDR := 0x47000000
+  IMAGES := sysupgrade.bin
+  IMAGE_SIZE := 15424k
+  SUPPORTED_DEVICES += R36
+  KERNEL := kernel-bin | lzma | \
+	fit lzma $$(KDIR)/image-$$(firstword $$(DEVICE_DTS)).dtb
```

Comment:
I tend to use gzip for the main firmware to (significantly) decrease the time it takes to decompress the kernel at the cost of slightly bigger size. Imho it's worth it. But up to you, I don't mind too much,

Reply:
lzma is probably 300KB smaller than gzip. On this device using lzma might be worth it.

Reply:
I'll leave it as it is . OEM fw use lzma as well https://forum.openwrt.org/t/add-support-for-cudy-re3000/187940?u=nicefile